item.rb

```diff
@@ -0,0 +1,17 @@
+class Item
+  attr_accessor :genre, :source, :author, :label
+
+  def initialize(publish_date, archived: false)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def can_be_archived?
+    true if @publish_date > '2012/09/05'
+  end
```

Comment:
- [ ] Kindly make sure that the `can_be_archived?` method returns `true` if the published date is older than 10 years and `false` in the opposite case. 
You would increase the functionality of the method if it works with the current date instead of a static date. Try using the `DateTime` [ruby subclass](https://ruby-doc.org/stdlib-2.6.1/libdoc/date/rdoc/DateTime.html#method-c-parse) to achieve this.